source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -237,6 +237,22 @@
             return valuesByKey;
         });
 
+        var commentsByKeyCollection = _.map(data.Califications, function (calification) {
+            var commentsByKey = {
+                commentRow: {
+                    calificationId: calification.Id,
+                    evaluatorEmployee: calification.EvaluatorEmployee,
+                    owner: calification.Owner
+                }
+            };
+            if (calification.Califications)
+                for (var calif in calification.Califications) {
+                    var cal = calification.Califications[calif];
+                    commentsByKey[cal.Key] = cal.Comment;
+                }
+            return commentsByKey
```

Comment:
I cannot understand what `commentsByKey` means. Could you help me? `commentRow` is one of the keys?